src/components/color-picker/color-picker.tsx

```diff
@@ -366,7 +366,7 @@ export class ColorPicker
 
   private handleChannelInput = (event: CustomEvent): void => {
     const input = event.currentTarget as HTMLCalciteInputElement;
-    const internalInput = event.detail.nativeEvent.target as HTMLInputElement;
+    const internalInput = input?.shadowRoot.querySelector("input") as HTMLInputElement;
```

Comment:
I think this is ok for now but lets create an issue and add a comment here with the issue to refactor this.

Reply:
Maybe the input needs an undocumented method to set the internal input value? That might be a safer way to go

Reply:
Good idea. I ended up going with an internal method that just syncs the internal input's `value` with `this.value`. Seems like the most stable option for now. Can I get a re-review if you're still on @driskull